### PR TITLE
Migrate TestCreateTmpfsMountsTarget test to api test

### PR DESCRIPTION
**Pull request by yongtang**
This fix migrates TestCreateTmpfsMountsTarget test to api test, and removed integration-cli/docker_cli_create_unix_test.go

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

File: integration/container/create_test.go [hunk 1]
```diff
@@ -91,3 +92,51 @@ func TestCreateWithInvalidEnv(t *testing.T) {
...
+		{
+			target:        "//",
+			expectedError: "destination can't be '/'",
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+		t.Run(tc.target, func(t *testing.T) {
+			t.Parallel()
```
Comment (cpuguy83) [inline integration/container/create_test.go]: Can't use parallel with the deferred cleanup.

**Review by cpuguy83 — CHANGES_REQUESTED** [review]

File: integration/container/create_test.go (hunk 1 above)
Comment (yongtang) [reply integration/container/create_test.go]: @cpuguy83 Update. Thanks.

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Review by dnephin — APPROVED** [review]
LGTM

**Pull request merged**